### PR TITLE
test(cdc-large-cluster): Multidc with cdc and large cluster

### PR DESCRIPTION
**Pull request by aleksbykov**
Added new test to check cdc behavior with large cluster
in case of verifying commit scylladb/scylla: 5f84848

Trello: https://trello.com/c/USmkUK46

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: test-cases/longevity/longevity-cdc-100gb-8h-multi-dc-large-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+nemesis_class_name: 'SisyphusMonkey'
+nemesis_include_filter: ['topology_changes']
+nemesis_seed: '023'
+nemesis_interval: 5
+nemesis_add_node_cnt: 3
+
+round_robin: true
+
+user_prefix: 'longevity-cdc-100gb-3h-multidc-large-cluster'
```
Comment (fgelcer) [inline test-cases/longevity/longevity-cdc-100gb-8h-multi-dc-large-cluster.yaml:23]: this doesn't fit the file name (or the file name doesn't fit this parameter

File: test-cases/longevity/longevity-cdc-100gb-8h-multi-dc-large-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+test_duration: 720
```
[the remark this answers is not in the document]
Comment (roydahan) [reply test-cases/longevity/longevity-cdc-100gb-8h-multi-dc-large-cluster.yaml:1]: The setup takes long time

**Review by roydahan — APPROVED** [review]

File: test-cases/longevity/longevity-cdc-100gb-8h-multi-dc-large-cluster.yaml (hunk 1 above)
Comment (roydahan) [reply test-cases/longevity/longevity-cdc-100gb-8h-multi-dc-large-cluster.yaml:23]: On one hand the stress is 3h, OTOH the job is set to 8h.
Not sure what's better, for now it's good enough.

**Comment by rayakurl** [comment]
@roydahan - please backport to 4.6. Thanks

**Comment by fruch** [comment]
@rayakurl @aleksbykov backported to 4.6
please create the jenkins as needed for 4.6

**Pull request merged**